src/app/consts.py

```diff
@@ -7,9 +7,13 @@
 # Images
 WATER_IMAGE_PATH = 'assets/water.png'
 GRASS_IMAGE_PATH = 'assets/grass.png'
-
+OUTLINE_IMAGE_PATH = 'assets/outline.png'
 SPLASH_IMAGE_PATH = "assets/splash.png"
 
+# Hex grid
+# size -> distance from middle of hex to a corner
+HEX_SIZE = 64.0
```

Comment:
any reason for float?

Reply:
Also, this assumes symmetric hexagons. might want to modify somewhere in the future